lib/xandra/authenticator.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Xandra.Authenticator do
+  @moduledoc """
+  A behaviour module for implementing a Cassandra authenticator
+
+  ## Example
+
+    defmodule MyAuthenticator do
+      @behaviour Xandra.Authenticator
+
+      def response_body(options) do
+        [Keyword.fetch(options, :foo), Keyword.fetch(options, :bar)]
```

Comment:
`Keyword.fetch/2` returns `{:ok, term}` so you probably want `Keyword.fetch!/2` here. I'd use a more verbose example here:

```elixir
def response_body(options) do
  ["user:", Keyword.fetch!(options, :user), "_password:", Keyword.fetch!(options, :password)]
end
```